### ISSUE TITLE
spaces+hyphens in names cause problems

### ISSUE THREAD

**Issue by mclarke** [infiton/capitalize-names#6]
```
[2] pry(main)> CapitalizeNames.capitalize('Cinnamon Ballantye - Clarke')
=> "Cinnamon Ballantye -larke"
```

**Comment by dewolfe** [comment]
+1

**Comment by infiton** [comment]
@mclarke @dewolfe this issue was fixed in #7 and has been released to rubygems https://rubygems.org/gems/capitalize-names/versions/1.0.6

update to `>=1.0.6` to solve your issues
